### PR TITLE
[BrokenLinksH2] Fixed broken links

### PR DESCRIPTION
**Pull request by kturetz**
Fixed broken links in iis-docs.

### CODE REVIEW

**Comment by kturetz** [comment]
**Global effort to fix broken links** 

@Rick-Anderson 
@rmcmurray 
@david-puglielli 

The Content & Learning team is fixing broken links on docs.microsoft.com for the rest of H2. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only link fixes and does not change other content. I’d very much appreciate your timely review and feedback on the link changes I’ve suggested. Thanks!

File: iis/application-frameworks/running-classic-asp-applications-on-iis-7-and-iis-8/classic-asp-parent-paths-are-disabled-by-default.md [hunk 1]
```diff
@@ -79,4 +79,4 @@ For additional information about the options that are available for classic ASP,
 For additional detail on parent paths in IIS, see the following page in the Microsoft Knowledge Base:
 
 > **Enable Parent Paths Is Disabled by Default in IIS 6.0**  
-> [https://support.microsoft.com/kb/332117](https://support.microsoft.com/kb/332117)
+> `https://support.microsoft.com/kb/332117`
```
Comment (wadepickett) [inline iis/application-frameworks/running-classic-asp-applications-on-iis-7-and-iis-8/classic-asp-parent-paths-are-disabled-by-default.md:82]: This link appears to be broken.  I didn't see a replacement to suggest.  Not sure where that article went.  Searching on the site for the kb number suggests there is a file that includes that keyword but says the site is not allowed to show it.

**Review by wadepickett — CHANGES_REQUESTED** [review]
Great to see a lot of these fixed.  
Some of the links that were addressed in changes seem to be out of date and broken.  I noted each.  I also corrected a few minor typos.  I'll commit the ones that have no pending question about them.

File: iis/application-frameworks/install-and-configure-php-applications-on-iis/openads-on-iis.md [hunk 2]
```diff
@@ -15,15 +15,15 @@ by Eric Woersching
...
 
 The following steps assume that you have completed the setup and configuration of FastCGI extension and PHP libraries as described in [https://www.iis.net/fastcgi/phpon60](using-fastcgi-to-host-php-applications-on-iis-60.md).
 
 ## Download and Unpack OpenAds
 
-First, download the latest stable release of the OpenAds application. For this article, we use openads-2.4.1 downloaded from the mirror [http://www.openads.org/products/openads/download?ext=zip&amp;utm\_source=openads&amp;utm\_medium=f&amp;utm\_campaign=upgrade241](http://www.openads.org/products/openads/download?ext=zip&amp;utm_source=openads&amp;utm_medium=f&amp;utm_campaign=upgrade241).
+First, download the latest stable release of the OpenAds application. For this article, we use openads-2.4.1 downloaded from the mirror `http://www.openads.org/products/openads/download?ext=zip&amp;utm\_source=openads&amp;utm\_medium=f&amp;utm\_campaign=upgrade241`.
```
[the remark this answers is not in the document]
Comment (kturetz) [reply iis/application-frameworks/install-and-configure-php-applications-on-iis/openads-on-iis.md:26]: Thanks for your detective work, Wade! Too bad they didn't just forward the old domain!

File: iis/application-frameworks/install-and-configure-php-applications-on-iis/openads-on-iis.md [hunk 3]
```diff
@@ -15,15 +15,15 @@ by Eric Woersching
 
 OpenAds 2.0 is an ad server written in PHP that allows you to place banners on your website and manage your banners in a professional way. OpenAds is open source, which means you have access to the source code, licensed under the General Public License. OpenAds is the latest version of PHPAds.
 
-For more information about OpenAds, see [http://www.openads.org/](http://www.openads.org/).
+For more information about OpenAds, see `http://www.openads.org/`.
```
[the remark this answers is not in the document]
Comment (kturetz) [reply iis/application-frameworks/install-and-configure-php-applications-on-iis/openads-on-iis.md:18]: How about:

For more information about Revive Adserver (formerly OpenAds), see `https://www.revive-adserver.com/`.

File: iis/application-frameworks/install-and-configure-php-applications-on-iis/openads-on-iis.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (kturetz) [reply iis/application-frameworks/install-and-configure-php-applications-on-iis/openads-on-iis.md:26]: How about this?

For this article, we use Revive Adserver (formerly OpenAds). See https://www.revive-adserver.com/.

File: iis/application-frameworks/install-and-configure-php-applications-on-iis/openads-on-iis.md [hunk 4]
```diff
@@ -50,7 +50,7 @@ After placing the files on the Web server, make sure the Web server can change t
 
 OpenAds 2.0 contains an automatic installer which does most of the work for you. Do not manually edit the configuration files, because that could cause problems in the future. Follow the guidance given below.
 
-1. To start the installer, open a Web browser and point it to the directory in which [http://localhost/openads](http://localhost/openads). This starts the installer automatically. If it does not start automatically, use [http://192.168.2.10/openads/www/admin/install.php](http://192.168.2.10/openads/www/admin/install.php).  
+1. To start the installer, open a Web browser and point it to the directory in which `http://localhost/openads`. This starts the installer automatically. If it does not start automatically, use `http://192.168.2.10/openads/www/admin/install.php`).
```
[the remark this answers is not in the document]
Comment (kturetz) [reply iis/application-frameworks/install-and-configure-php-applications-on-iis/openads-on-iis.md:53]: This is getting a little tricky. If permissible, I'd comment out the links and add a paragraph:

For more information about Revive Adserver (formerly OpenAds), see https://www.revive-adserver.com/.

File: iis/application-frameworks/install-and-configure-php-applications-on-iis/openads-on-iis.md (hunk 2 above)
Comment (wadepickett) [inline iis/application-frameworks/install-and-configure-php-applications-on-iis/openads-on-iis.md:26]:
```suggestion
First, download the latest stable release of the OpenAds application. For this article, we use Revive Adserver (formerly OpenAds). See [https://www.revive-adserver.com/](https://www.revive-adserver.com/).
```

File: iis/application-frameworks/install-and-configure-php-applications-on-iis/openads-on-iis.md (hunk 4 above)
Comment (wadepickett) [inline iis/application-frameworks/install-and-configure-php-applications-on-iis/openads-on-iis.md:53]:
```suggestion
1. To start the installer, open a Web browser and point it to the directory in which `http://localhost/openads`. This starts the installer automatically. For more information about Revive Adserver (formerly OpenAds), see [https://www.revive-adserver.com/](https://www.revive-adserver.com/). 
```

File: iis/application-frameworks/install-and-configure-php-on-iis/secure-your-sql-server-database.md [hunk 5]
```diff
@@ -171,10 +171,10 @@ The following table contains information about security views and functions.
...
+- SQL Server 2008 Deployment Guide for Web Hosting Environments `https://download.microsoft.com/download/d/1/f/d1f05c2e-0741-442d-859e-9417b377fdfa/The%20architectural%20view%20of%20SQL%20Server.zip`
 - [Security Considerations for a SQL Server Installation](https://msdn.microsoft.com/library/ms144228.aspx)
 - [Secure Deployment (Database Engine)](https://msdn.microsoft.com/library/bb522545.aspx)
 - [Security and Protection (Database Engine)](https://msdn.microsoft.com/library/bb510589.aspx)
 - [Security Enhancements in Microsoft SQL Server 2008 video](https://mediadl.microsoft.com/mediadl/IISNET/Media/Security%20Enhancements%20in%20Microsoft%20SQL%20Server%202008.wmv)
-- [Security Enhancements in Microsoft SQL Server 2008 presentation](https://download.microsoft.com/download/d/1/f/d1f05c2e-0741-442d-859e-9417b377fdfa/Security%20Enhancements%20in%20SQL%20Server%202008.zip)
+- Security Enhancements in Microsoft SQL Server 2008 presentation `https://download.microsoft.com/download/d/1/f/d1f05c2e-0741-442d-859e-9417b377fdfa/Security%20Enhancements%20in%20SQL%20Server%202008.zip`
```
Comment (wadepickett) [inline iis/application-frameworks/install-and-configure-php-on-iis/secure-your-sql-server-database.md:179]:
```suggestion
```

File: iis/application-frameworks/running-classic-asp-applications-on-iis-7-and-iis-8/classic-asp-parent-paths-are-disabled-by-default.md (hunk 1 above)
Comment (wadepickett) [inline iis/application-frameworks/running-classic-asp-applications-on-iis-7-and-iis-8/classic-asp-parent-paths-are-disabled-by-default.md:82]:
```suggestion
```

**Review by wadepickett — APPROVED** [review]
Updated per feedback and approved.

File: iis/application-frameworks/install-and-configure-php-on-iis/secure-your-sql-server-database.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (wadepickett) [reply iis/application-frameworks/install-and-configure-php-on-iis/secure-your-sql-server-database.md]: Removed link.

File: iis/application-frameworks/running-classic-asp-applications-on-iis-7-and-iis-8/classic-asp-parent-paths-are-disabled-by-default.md (hunk 1 above)
Comment (wadepickett) [reply iis/application-frameworks/running-classic-asp-applications-on-iis-7-and-iis-8/classic-asp-parent-paths-are-disabled-by-default.md]: Removed link.

File: iis/application-frameworks/install-and-configure-php-on-iis/secure-your-sql-server-database.md [hunk 6]
```diff
@@ -171,10 +171,9 @@ The following table contains information about security views and functions.
 ## Links for Further Information
 
 - [Securing SQL Server](https://msdn.microsoft.com/library/bb283235.aspx)
-- [SQL Server 2008 Deployment Guide for Web Hosting Environments](https://download.microsoft.com/download/d/1/f/d1f05c2e-0741-442d-859e-9417b377fdfa/The%20architectural%20view%20of%20SQL%20Server.zip)
+- SQL Server 2008 Deployment Guide for Web Hosting Environments `https://download.microsoft.com/download/d/1/f/d1f05c2e-0741-442d-859e-9417b377fdfa/The%20architectural%20view%20of%20SQL%20Server.zip`
```
Comment (wadepickett) [inline iis/application-frameworks/install-and-configure-php-on-iis/secure-your-sql-server-database.md:174]:
```suggestion
```

Comment (wadepickett) [reply iis/application-frameworks/install-and-configure-php-on-iis/secure-your-sql-server-database.md:174]: No longer available.  Removing.

**Comment by kturetz** [comment]
Great work, @wadepickett! Are you able to merge this PR?

**Pull request merged**